ios/RNGoogleSignin/RNGoogleSignin.m

```diff
@@ -230,12 +227,11 @@ - (void)signIn:(GIDSignIn *)signIn dismissViewController:(UIViewController *)vie
   [viewController dismissViewControllerAnimated:true completion:nil];
 }
 
-+ (BOOL)application:(UIApplication *)application openURL:(NSURL *)url
-  sourceApplication:(NSString *)sourceApplication annotation: (id)annotation {
-
-  return [[GIDSignIn sharedInstance] handleURL:url
-                             sourceApplication:sourceApplication
-                                    annotation:annotation];
++ (BOOL)application:(UIApplication *)application
```

Comment:
@D1verGW one more thing, can you please replace this by the newer method?

https://developer.apple.com/documentation/uikit/uiapplicationdelegate/1623073-application?language=objc is deprecated, thanks

Reply:
I'll handle it myself, thanks